### PR TITLE
Back button feat

### PR DESCRIPTION
**Pull request by Allislove**

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -100,3 +104,25 @@ h1 {
...
+  align-items: center;
+  justify-content: center;
+  margin: 0.5rem 0;
+  padding: 0.5rem;
+}
+
+#surveyForm > div:nth-child(2) button {
+  padding: 0.5rem;
+  margin: 0 0.25rem;
+
```
Comment (Israel-Laguan) [inline src/styles/globals.css]: Remove this extra line

File: src/pages/exam.tsx [hunk 2]
```diff
@@ -141,13 +150,25 @@ const Exam = () => {
...
+            {currentQuestionIndex > 0 ? (
+              <button onClick={handlePreviousQuestion} type="submit">
+                Anterior
+              </button>
+            ) : null}
+            {questions.length > 0 && currentQuestionIndex < questions.length - 1 ? (
+              <button onClick={handleNextQuestion} type="submit">
+                Siguiente
+              </button>
+            ) : null}
```
Comment (Israel-Laguan) [inline src/pages/exam.tsx]: Instead of this null only use shortcircuit operator instead (&&)

**Comment by Allislove** [comment]
@Israel-Laguan  Can you review again plz.

**Comment by Israel-Laguan** [comment]
image
Looks better

But I noticed the option is not saved, for example

1- choose an option
image

2- advance
Select an option on the next question
image

3- come back. Notice the previous answer was not saved
image

4- answer then advance, notice the previous answer was not saved
image

**Comment by Allislove** [comment]
> But I noticed the option is not saved, for example

Ok, can you review again please.

File: src/pages/exam.tsx [hunk 3]
```diff
@@ -119,16 +135,21 @@ const Exam = () => {
       {loading ? <Loader /> : null}
       {error ? error : <p>{success}</p>}
       {questions.length > 0 && currentQuestionIndex < questions.length ? (
-        <form id="surveyForm" onSubmit={handleNextQuestion}>
+        <form id="surveyForm">
           {success ? success : error}
           <div key={currentQuestionIndex}>
             <ul>{questions[currentQuestionIndex].question}</ul>
-            {questions[currentQuestionIndex].options.map((option: any, k2) => (
+            {questions[currentQuestionIndex].options.map((option: any, k2: any) => (
```
Comment (Israel-Laguan) [inline src/pages/exam.tsx]: K2 is a number always, I think

File: src/pages/exam.tsx [hunk 4]
```diff
@@ -141,13 +162,25 @@ const Exam = () => {
...
+              </button>
+            )}
+          </div>
         </form>
-      ) : (
-        <button onClick={surveyEnd} type="button">
-          Finalizar
-        </button>
-      )}
+      ) : null}
```
Comment (Israel-Laguan) [inline src/pages/exam.tsx]: Maybe this null in not needed if you use short-circuit operator (&&)

File: src/pages/exam.tsx [hunk 5]
```diff
@@ -141,13 +162,25 @@ const Exam = () => {
               </li>
             ))}
           </div>
-          <button type="submit">Siguiente</button>
+          <div>
+            {currentQuestionIndex > 0 && (
+              <button onClick={handlePreviousQuestion} type="submit">
+                Anterior
```
Comment (Israel-Laguan) [inline src/pages/exam.tsx]: Please put everything in the UI in English

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/exam.tsx (hunk 4 above)
Comment (Allislove) [reply src/pages/exam.tsx]: Done

File: src/pages/exam.tsx (hunk 5 above)
Comment (Allislove) [reply src/pages/exam.tsx]: Done

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**